functions-app/calculate-score-v2/school.data.service.js

```diff
@@ -0,0 +1,13 @@
+const R = require('ramda')
+const sqlService = require('../lib/sql/sql.service')
+
+/**
+ * Get all school ids
+ * @return {Promise<object>}
+ */
+module.exports.sqlFindSchoolIds = async () => {
```

Comment:
Shouldn't this be in the school scores data service?  It's part of that vertical slice.

Reply:
@jon-shipley it's a bit trivial in my opinion but moved anyway